rest_framework_json_api/views.py

```diff
@@ -129,7 +129,10 @@ def get_queryset(self, *args, **kwargs):
 
 class RelatedMixin:
     """
-    This mixin handles all related entities, whose Serializers are declared in "related_serializers"
+    Mixing handling related links.
```

Comment:
I don't understand what this added comment is about

Reply:
I only reformatted that comment into a proper docstring to satisfy bugbear. The comment is really not that clear and helpful, though. Added it to my list TODO to change it (most likely with a general docstring update).